### PR TITLE
Added Getting started chapter

### PR DESCRIPTION
**Pull request by breakreturn**
Updated user docs with a getting started chapter.
Also added comment to containerabstractinterface

### CODE REVIEW

File: doc/chapters/getting-started/index.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+From the build directory, run the agent::
+
+    sudo ./agent/softwarecontainer-agent
+
+Not that we are running this command with `sudo`, the Agent needs to be started with root privileges.
+
+
+Using `dbus-send`, introspect the Agent D-Bus service API::
+
+    dbus-send --system --print-reply --dest=com.pelagicore.SoftwareContainerAgent /com/pelagicore/SoftwareContainerAgent org.freedesktop.DBus.Introspectable.Introspect
```
Comment (DunderRoffe) [inline doc/chapters/getting-started/index.rst]: I think you should split this over several lines to make it easier and faster to read.
Using '\' will persevere the "copy and paste"-ability if that is a concern.
Eg:

``` bash
dbus-send --system --print-reply \
    --dest=com.pelagicore.SoftwareContainerAgent \
      /com/pelagicore/SoftwareContainerAgent \
      org.freedesktop.DBus.Introspectable.Introspect
```

**Comment by DunderRoffe** [comment]
Besides the note about multi line over long single line above, it looks really good :)

File: doc/chapters/getting-started/index.rst (hunk 1 above)
Comment (tobsan) [reply doc/chapters/getting-started/index.rst]: +1

File: doc/chapters/getting-started/index.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+Prerequisites:
+  * configure and build (see project `README.md`)
+
+
+From the build directory, run the agent::
+
+    sudo ./agent/softwarecontainer-agent
+
+Not that we are running this command with `sudo`, the Agent needs to be started with root privileges.
```
Comment (tobsan) [inline doc/chapters/getting-started/index.rst]: Note*

**Comment by breakreturn** [comment]
Updated now based on feedback.

**Pull request merged**